### DISCUSSION TITLE
S23FE, ALWAYS 60HZ IN PSM MODE EVEN I TURNED ON THE STUFF

### DISCUSSION THREAD

**Discussion by Roychan10247** [tribalfs/GalaxyMaxHzPub#235] [General]
HELP
![Screenshot_20240911_211332_Galaxy MaxHz](https://github.com/user-attachments/assets/ac421260-823d-4d5d-a1a0-e06f772fa585)

**Comment by tribalfs** [comment]
Click and read/follow.

![image](https://github.com/user-attachments/assets/6c44746d-3775-41a1-acf2-df8b2fb37ada)
